validate.go

```diff
@@ -194,6 +194,19 @@ func FormatFieldError(key string, fieldErr *validator.FieldError, root *RootConf
 	}
 
 	switch fieldErr.Tag {
+	case "int":
+		return fmt.Errorf("Value %q must be a valid integer at key %q", fieldErr.Value, formatted)
+
+	case "configitemtype":
+		var validTypes []string
+		for t := range configItemValidTypes {
```

Comment:
Is this a map being converted to an array?

Reply:
map[string]bool -> []string